kratos/python_scripts/analysis_stage.py

```diff
@@ -411,6 +416,96 @@ def __CreateListOfProcesses(self):
         self._list_of_processes.extend(self._list_of_output_processes) # Adding the output processes to the regular processes
         self._list_of_output_processes.extend(deprecated_output_processes)
 
+    def __AddProcessesVariables(self):

```

Comment:
jesus what a monster :sweat_smile: 

Can you please move this into a separate file, maybe with the process-factory?

Reply:
Sure. I will do if this becomes the final implementation. So far is something on top which the @KratosMultiphysics/technical-committee can discuss. :wink: